factory/.env

```
@@ -11,7 +11,7 @@ BASE_IMAGE='debian:bullseye-slim'
 NODE_VERSION='18.14.1'
 
 # Update this to deploy the docker factory if you make changes to factory.Dockerfile or install scripts
-FACTORY_VERSION='2.0.0'
+FACTORY_VERSION='2.0.1'
 
 # Chrome versions: https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable
 CHROME_VERSION='111.0.5563.64-1'
```

weirdly it seems like chrome is installing a different version that requested, but thats a problem for another pr....

```suggestion
CHROME_VERSION='111.0.5563.110-1'
```